packages/components/src/button/style.scss

```diff
@@ -194,7 +194,41 @@
 
 	// Link buttons that are red to indicate destructive behavior.
 	&.is-link.is-destructive {
-		color: $alert-red;
+		color: #b52727;
```

Comment:
This is a new color, which is fine, but we should change the variable contents instead of adding a new color, that way all elements that reference $alert-red will benefit from the change, and the color will be consistent.

This picture shows the existing $alert-red in the top left corner, the new color you added in the bottom, and a suggestion on the right. 

![Group 50](https://user-images.githubusercontent.com/1204802/83874092-3acf7080-a735-11ea-9cf6-171be81573f1.png)

The existing alert red is a little soft. Your new color is better, but I wonder if there's a 3rd option, which I suggested on the right, it's a bit between them and with higher saturation. What do you think? That new color is #CC1818